en/developer_guide/routing.rst

```diff
@@ -102,27 +114,30 @@ Declaring differents loaders as services in ``routing.yml`` is mandatory to gene
         resource: '.'
         type: orchestra_redirection
 
-See `Symfony Loader documentation`_ for more information.
+You also have to modify the project configuration, in the ``config.yml`` file :
+
+.. code-block:: yaml
+
+    open_orchestra_front:
+        routing_type: file
+
+How to choose between both of the solution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this part, I would not tell you which routing type you should use, but only list a some condition to use
+one or the other.
 
-URL Generator
-~~~~~~~~~~~~~
+You are not using the Open Orchestra Backoffice :
 
-``OpenOrchestraUrlGenerator`` generates URLs from routes.
-The URL Generator is overridden to take `RequestStack`_ in argument and get the ``aliasId`` from it.
-It will be used to adapt this route name with route names generated by Open Orchestra ``Route Loader``.
+* Use the file routing, or you will have to manually update the ``route_document`` collection
```

Comment:
I'm not sure this condition is enough as there is a serious performance issue with this router. For instance if there's no standard back office but an api updating frequently the nodes, the front office will probably collapse with this router.
So I'll say "You are not using the Open Orchestra Backoffice and you have a low node contribution rate"